libass/ass_types.h

```diff
@@ -207,6 +207,9 @@ typedef struct ass_track {
     ASS_Library *library;
     ASS_ParserPriv *parser_priv;
 
+    char *process_data_buf;
+    size_t process_data_buf_size;
```

Comment:
I think it's possible to use `ASS_StringView process_data_buf;` for that.

Reply:
`ASS_StringView` is an internal struct; `ass_types.h` is a public header, so `ASS_StringView` can't be used here.
However, I think it's possible and would be better to move the bufffer into `parser_priv`.

Reply:
Agreed; moved there.